templates/zerver/help/configure-authentication-methods.md

```diff
@@ -0,0 +1,18 @@
+# Configure authentication methods
+
+{!admin-only.md!}
+
+{!go-to-the.md!} [Authentication methods](/#organization/auth-methods)
+{!admin.md!}
+
+2. Enable or disable the authentication methods you want.
```

Comment:
That's not all the methods Zulip offers (see [here](http://zulip.readthedocs.io/en/latest/production/authentication-methods.html)) indicates that there are other methods (such as LDAP), not shown in the modal you took. You should explain how to enable those methods by modifying the settings configuration file.